OpenSim/Simulation/Wrap/WrapTorus.cpp

```diff
@@ -69,50 +65,17 @@ WrapTorus::~WrapTorus()
 {
 }
 
-//_____________________________________________________________________________
-/**
- * Copy constructor.
- *
- * @param aWrapTorus WrapTorus to be copied.
- */
-WrapTorus::WrapTorus(const WrapTorus& aWrapTorus) :
-    WrapObject(aWrapTorus),
-   _innerRadius(_innerRadiusProp.getValueDbl()),
-   _outerRadius(_outerRadiusProp.getValueDbl())
-{
-    setNull();
-    setupProperties();
-    copyData(aWrapTorus);
-}
-
-
 //=============================================================================
 // CONSTRUCTION METHODS
 //=============================================================================
-//_____________________________________________________________________________
-/**
- * Set the data members of this WrapTorus to their null values.
- */
-void WrapTorus::setNull()
-{
-}
-
 //_____________________________________________________________________________
 /**
  * Connect properties to local pointers.
  */
-void WrapTorus::setupProperties()
+void WrapTorus::constructProperties()
 {
-    // BASE CLASS
-    //WrapObject::setupProperties();
-
-    _innerRadiusProp.setName("inner_radius");
-    _innerRadiusProp.setValue(-1.0);
-    _propertySet.append(&_innerRadiusProp);
-
-    _outerRadiusProp.setName("outer_radius");
-    _outerRadiusProp.setValue(-1.0);
-    _propertySet.append(&_outerRadiusProp);
+    constructProperty_inner_radius(1.0);
+    constructProperty_outer_radius(1.0);
```

Comment:
Consider using more reasonable defaults (inner radius of 0.05 cm, outer radius of 0.01 cm).

Reply:
Oh I meant 0.05 m and 0.01 m. thanks for checking.